examples/duel/www/index.html

```diff
@@ -1,17 +1,18 @@
 <!DOCTYPE html>
-<html lang="en" data-framework="duel">
+<html lang="en">
 <head>
-	<meta charset="utf-8" />
-	<title>DUEL &#x2022; TodoMVC</title>
-	<link rel="stylesheet" href="./cdn/bada2673af7edc043a573c3e20ac2d8ce9d10037.css" />
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>DUEL &bull; TodoMVC</title>
+	<link rel="stylesheet" href="./cdn/e9c3940d189473aceb2b4bc8cc436b995257df0b.css">
 </head>
 <body>
-	<footer id="info">
+	<footer class="info">
 		<p>Double-click to edit a todo</p>
 		<p>Ported to <a href="http://duelengine.org">DUEL</a> by <a href="http://mck.me">Stephen McKamey</a></p>
 		<p>Part of <a href="http://todomvc.com">TodoMVC</a></p>
 	</footer>
-	<script src="./cdn/d10574a720ccdf2d2c810f0c45afb5f159264a1f.js"></script>
+	<script src="./cdn/916ee1b55ac72cc1cc01404ca8e13a695228cc13.js"></script>
 
 </body>
 </html>
```

Comment:
Again index.html is a generated file and `examples/duel/src/main/resoruces/HomePage.duel` is the source.